config_library/pattern-2/fcc-invoices/README.md

```diff
@@ -0,0 +1,354 @@
+- Deploys with FCC-specific prompts and schema
+
+**Expected output:**
+- Stack creation takes ~5-10 minutes
+- Stack status: `CREATE_COMPLETE`
+
+## Step 2: Run Inference
+
+Run inference on the sample documents:
+
+```bash
+idp-cli run-inference \
+  --stack-name fcc-inf-test \
+  --manifest samples/fcc-invoices/fcc_invoices_sample_3.csv \
+  --region us-west-2
+```
+
+**What this does:**
+- Uploads documents to S3 input bucket
+- Triggers Lambda processing pipeline
+- Performs OCR with Textract
+- Extracts structured data using Claude
+- Stores results in S3 output bucket
+
+**Expected output:**
+```
+Validating manifest...
+✓ Manifest validated successfully
+Initializing batch processor for stack: fcc-inf-test
+✓ Batch submitted successfully
+Batch ID: batch-20251017-140000
+Processing 3 documents...
+```
+
+**Monitor progress:**
+```bash
+idp-cli status \
+  --stack-name fcc-inf-test \
+  --batch-id <batch-id> \
+  --region us-west-2 \
+  --wait
+```
+
+## Step 3: Download Results
+
+Download the inference results locally:
+
+```bash
+idp-cli download-results \
+  --stack-name fcc-inf-test \
+  --batch-id cli-batch-20251017-190516 \
+  --output-dir fcc_results \
+  --region us-west-2
+```
+
+**Note**: Replace `cli-batch-20251017-190516` with your actual batch ID from the inference step. You can specify any output directory name.
+
+**What this does:**
+- Downloads all result files from S3
+- Creates directory structure: `fcc_results/<doc_id>/sections/1/result.json`
+- Each result contains extracted fields and metadata
+
+**Result structure:**
+```json
+{
+  "document_class": {
+    "type": "FCC-Invoice"
+  },
+  "inference_result": {
+    "agency": "Agency Name",
+    "advertiser": "Advertiser Name",
+    "gross_total": "1,234.56",
+    "net_amount_due": "1,234.56",
+    "line_item__description": ["M-F 11a-12p", "M-F 12n-1p"],
+    "line_item__days": ["MTWTF--", "MTWTF--"],
+    "line_item__rate": ["100.00", "150.00"],
+    "line_item__start_date": ["11/01/21", "11/01/21"],
+    "line_item__end_date": ["11/07/21", "11/07/21"]
+  }
+}
+```
+
+## Step 4: Run Evaluation
+
+Evaluate the extraction results against ground truth using the **simplified evaluation script** (recommended):
+
+```bash
+cd config_library/pattern-2/fcc-invoices
+
+python bulk_evaluate_fcc_invoices_simple.py \
+  --results-dir ../../../fcc_results/cli-batch-20251017-190516 \
+  --csv-path sample_labels_3.csv \
+  --config-path stickler_config.json \
+  --output-dir evaluation_output
+```
+
+**Alternative**: Use the legacy script (more complex, same results):
+```bash
+python bulk_evaluate_fcc_invoices.py \
+  --results-dir ../../../fcc_results/cli-batch-20251017-190516 \
+  --csv-path sample_labels_3.csv \
+  --config-path stickler_config.json \
+  --output-dir evaluation_output
+```
+
+**Note**: The `sample_labels_3.csv` contains ground truth for 3 sample documents. For full dataset evaluation, use `sr_refactor_labels_5_5_25.csv`.
```

Comment:
Hmm.. So your ground truth format isn't the same as the inference results format?
How will this work in the accelerator for HITL review -> Save as Evaluation Baseline - which basically copies the verified inference results to be used as ground truth in future..  Let's discuss how to align ground truth format  with inference results format